.github/workflows/tests.yml

```diff
@@ -53,5 +54,5 @@ jobs:
       - name: "Validate Composer dependencies"
         run: "composer validate"
 
-      - name: "Run QC scripts"
-        run: "composer qc"
+      - name: "Run Test suite"
+        run: "composer test"
```

Comment:
What do you think about running static analysis and other tools against locked dependencies?

Reply:
I think we should look at static analysis in its own workflow personally

Reply:
Sounds good